include/swift/Basic/Features.def

```diff
@@ -117,6 +117,7 @@ UPCOMING_FEATURE(BareSlashRegexLiterals, 354, 6)
 UPCOMING_FEATURE(ExistentialAny, 335, 6)
 UPCOMING_FEATURE(ImportObjcForwardDeclarations, 384, 6)
 UPCOMING_FEATURE(DisableOutwardActorInference, 401, 6)
+UPCOMING_FEATURE(InternalImports, 409, 6)
```

Comment:
This feature name makes it sound like you need to specify it in order to write `internal import`. What about something like `InternalImportsByDefault`?

Reply:
Good point, I've updated the PR to use `InternalImportsByDefault`.